src/read.jl

```diff
@@ -143,6 +145,10 @@ function read_demand_flexibility(filepath, interval)::DemandFlexibility
                 "The parameter that indicates if the rolling load balance constraint " *
                 "is enabled is not defined. Will default to being enabled."
             ),
+            "source" => (
+                "The parameter that indicates the source of demand flexibility amount " *
+                "is not defined. Will default to DOE data."
+            )
```

Comment:
I know the current implementation involves passing the DOE data into `REISE.jl` via profiles, but once we go to the planned method where the DOE data is either in blob storage or hosted locally, we might want to rethink this default. For instance, if the source parameter is not provided but demand flexibility profiles are provided, then the source should probably be set to `"user"`. Alternatively, if the source parameter is not provided, demand flexibility profiles are not provided, and demand flexibility is enabled, then the source should probably be set to `"doe"`.